features/org.eclipse.emf.mwe2.launcher-feature/feature.xml

```diff
@@ -35,7 +35,7 @@ http://www.eclipse.org/legal/epl-v10.html
       <import plugin="org.eclipse.xtext.common.types" version="2.10.0" match="greaterOrEqual"/>
       <import plugin="org.eclipse.xtext.xbase" version="2.10.0" match="greaterOrEqual"/>
       <import plugin="com.google.inject" version="3.0.0" match="greaterOrEqual"/>
-      <import plugin="com.google.guava" version="10.0.0" match="greaterOrEqual"/>
+      <import plugin="com.google.guava" version="14.0.0" match="greaterOrEqual"/>
```

Comment:
I wished for a newer guava version as the lower bound. And an even newer version as the upper bound.

Reply:
yes but that is a separate issue. i just changed this to the range we have in xtext.
orbit plans to obsolte guava 15 and 19.
if we want to have a newer version i think we have to do the "paperwork"